openfisca_core/formulas.py

```diff
@@ -639,14 +635,15 @@ def calculate_output_add(formula, period):
     return formula.holder.compute_add(period).array
 
 
-def calculate_output_add_divide(formula, period):
-    return formula.holder.compute_add_divide(period).array
-
-
 def calculate_output_divide(formula, period):
     return formula.holder.compute_divide(period).array
 
 
+def calculate_output_first_month(formula, period):
```

Comment:
This is needed to compute a monthly variable for a year.

The yaml tests compute variables for the (implicit) simulation period. This period may not match the variable period unit, requiring to add a hook calculate_output. The need to remove the simulation period is already stated in #460 .